rusty_pizza_server/src/order_model/user.rs

```diff
@@ -0,0 +1,33 @@
+#[derive(Debug, PartialEq, Eq, Hash)]
+pub struct User {
+    name: String,
+}
+
+impl User {
+    pub fn new(name: String) -> User {
+        User { name }
+    }
+
+    pub fn set_name(&mut self, name: String) {
+        self.name = name
+    }
+
+    pub fn get_name(&self) -> String {
+        String::from(self.name.as_str())
+    }
```

Comment:
Why did you add the getter and setter here?

Reply:
Because the name is no longe public, so you won´t be able to access it

Reply:
If we say an user is not able to change his name, we can remove `set_name()`.

Reply:
Yes, as the user is identified by its name making it mutable is a bad idea.